packages/example-studio/migrations/renameField.js

```diff
@@ -0,0 +1,43 @@
+import baseClient from 'part:@sanity/base/client'
```

Comment:
Perhaps start by explaining what the script does? What is the source and target fields?

Reply:
Done 👇